src/sections/Resources/Resources-grid/filters.js

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from "react";
+import { HiOutlineChevronUp, HiOutlineChevronDown } from "react-icons/hi";
+import { options } from "./options";
+
+import ResourceNavigationWrapper from "./filters.style";
+
+const Navigation = (props) => {
+    const [expandType, setExpandType] = useState(true);
+    const [expandProduct, setExpandProduct] = useState(false);
+    const [expandTech, setExpandTech] = useState(false);
+    const [expandMesh, setExpandMesh] = useState(false);
+
+    const data = React.useMemo(() => options);
+    let typeOptions = data.filter((data) => data.category === "Type");
+    let productOptions = data.filter((data) => data.category === "Product");
+    let techOptions = data.filter((data) => data.category === "Technology");
+    let meshOptions = data.filter((data) => data.category === "Service Mesh");
```

Comment:
If I'm not wrong this portion here is not required right? We can get this from the ResourceList.js file?? I might be wrong though

Reply:
This is required but I need to find a better way to do it. I was mapping over the options file just once but that was messing with the collapsible so I left it this way for now. I'll look into it though.